composer.json

```
@@ -24,20 +24,20 @@
     "require": {
         "php": ">=8.0",
         "bref/bref": "^1.2|^2.0",
-        "symfony/filesystem": "^5.2|^6.0",
-        "symfony/http-kernel": "^5.2|^6.0",
+        "symfony/filesystem": "^5.2|^6.0|^7.0",
+        "symfony/http-kernel": "^5.2|^6.0|^7.0",
         "symfony/psr-http-message-bridge": "^2.1",
```

Symfony changed the version strategy -> https://packagist.org/packages/symfony/psr-http-message-bridge
```suggestion
        "symfony/psr-http-message-bridge": "^2.1|^6.4|^7.0",
```